client/src/components/StreamField/StreamField.scss

```diff
@@ -0,0 +1,82 @@
+@import '../../../../node_modules/react-streamfield/src/index';
+
+
+.children-container {
```

Comment:
@BertrandBordage could you explain how this code relates to the styles defined in `react-streamfield/src/index`?

Reply:
This code is specific to Wagtail. Wagtail has some global CSS that override `react-streamfield`’s assumptions on the base styling loaded in the page. This SCSS fixes conflicts with Wagtail’s CSS, and also adds some Wagtail-specific styling, like the help text that should be in blocks.